ci/deploy/npm.py

```diff
@@ -99,11 +101,22 @@ def release_package(version: NpmPackageVersion, package_path: Path) -> None:
                 dist_tag = None
         logger.info("Releasing %s %s", name, version.node)
         set_npm_credentials(package_path)
-        extra_args = ["--tag", dist_tag] if dist_tag is not None else []
+        # If we do not specify a dist tag, this automatically is tagged as
+        # `latest`. So, force a dist tag for release builds that are lower than
+        # the stable version. This usually happens when we cut a hotfix release
+        # for the in-production version after a release has been cut for the
+        # next version.
+        extra_args = ["--tag", dist_tag or branch_tag]
```

Comment:
You don't really need extra_args anymore, can just add this to the spawn.runv call in the next line

Reply:
If dist_tag is None, then branch_tag will not even be set. I think dist_tag should be of type `str`, not `str | None`.

Reply:
>  You don't really need extra_args anymore, can just add this to the spawn.runv call in the next line

Yep. Updated.

> If dist_tag is None, then branch_tag will not even be set.

Not quite: if `dist_tag` is `latest` _but_ the version being published is lower than the latest version available on NPM, we set `dist_tag` to `None` (line 102) but keep the `branch_tag` set to `latest-{major}.{minor}`.